chart/templates/grafana-datasources-sec.yaml

```diff
@@ -61,12 +62,16 @@ stringData:
         database: {{ ternary (include "tobs.dburi.dbname" . ) ($kubePrometheus.grafana.timescale.datasource.dbName) ($isDBURI) }}
         editable: true
         secureJsonData:
-          password: {{ ternary (include "tobs.dburi.password" . ) ($kubePrometheus.grafana.timescale.datasource.pass) ($isDBURI) }}
+          {{ if ne ($passwd) "" -}}
+          password: {{ $passwd }}
+          {{- else -}}
+          password: ${GRAFANA_PASSWORD}
```

Comment:
Could you change the name of this variable so it is more evident that it holds the password for the datasource and not to grafana UI?

Reply:
Yeah  good idea.  Just changed it to make it a bit better.